### PR TITLE
Add dutch book review dataset

### PR DESCRIPTION
**Pull request by benjaminvdb**
- Name: Dutch Book Review Dataset (DBRD)
- Description: The DBRD (pronounced dee-bird) dataset contains over 110k book reviews along with associated binary sentiment polarity labels and is intended as a benchmark for sentiment classification in Dutch.
- Paper: https://arxiv.org/abs/1910.00896
- Data: https://github.com/benjaminvdb/DBRD
- Motivation: A large (real-life) dataset of Dutch book reviews and sentiment polarity (positive/negative), based on the associated rating.

Checks
- [x] Create the dataset script /datasets/dbrd/dbrd.py using the template
- [x] Fill the _DESCRIPTION and _CITATION variables
- [x] Implement _info(), _split_generators() and _generate_examples()
- [x] Make sure that the BUILDER_CONFIGS class attribute is filled with the different configurations of the dataset and that the BUILDER_CONFIG_CLASS is specified if there is a custom config class.
- [x] Generate the metadata file dataset_infos.json for all configurations
- [x] Generate the dummy data dummy_data.zip files to have the dataset script tested and that they don't weigh too much (<50KB)
- [x] Add the dataset card README.md using the template : fill the tags and the various paragraphs
- [x] Both tests for the real data and the dummy data pass.

### CODE REVIEW

File: datasets/dbrd/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  volume    = {abs/1910.00896},
+  year      = {2019},
+  url       = {http://arxiv.org/abs/1910.00896},
+  archivePrefix = {arXiv},
+  eprint    = {1910.00896},
+  timestamp = {Fri, 04 Oct 2019 12:28:06 +0200},
+  biburl    = {https://dblp.org/rec/journals/corr/abs-1910-00896.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+```
```
Comment (lhoestq) [inline datasets/dbrd/README.md:170]: Looks like the `Considerations for Using the Data` section has been removed

Could you re-add it please ? you can leave the content as `[More Information Needed]`

File: datasets/dbrd/README.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+Non-Dutch reviews were filtered out using [langdetect](https://github.com/Mimino666/langdetect), and all reviews should therefore be in Dutch (nl). They are written by reviewers on [Hebban](https://www.hebban.nl), a Dutch website for book reviews. Reviews aren't curated and are of varying lengths and qualities, as can be expected.
+
+## Dataset Structure
+
+### Data Instances
+
+The dataset contains three subsets: train, test and unsupervised. The `train` and `test` sets contain labels, while the `unsupervised` set doesn't (the value for label is -1 for each instance in unsupervised).
+
+```
+DatasetDict({
```
Comment (lhoestq) [inline datasets/dbrd/README.md:77]: here we expect to see one example of instance.
For example you can load the dataset and get one by running
```python
dataset["train"][0]
```

**Review by lhoestq — COMMENTED** [review]
Really cool thanks !

I left some (minor) comments

File: datasets/dbrd/README.md (hunk 1 above)
Comment (benjaminvdb) [reply datasets/dbrd/README.md:174]: I've re-added the `Considerations for Using the Data` section and provided two of its subsections. Only information `Discussion of Biases` is missing now and this is indicated with `[More Information Needed]`.

File: datasets/dbrd/README.md (hunk 2 above)
Comment (benjaminvdb) [reply datasets/dbrd/README.md]: I've now provided an example for this section.

**Comment by benjaminvdb** [comment]
> Really cool thanks !
> 
> I left some (minor) comments

Thank you for your comments! 👍 I went ahead and improved the dataset card using your suggestions and some tweaks of my own. I hope you like it! 😄

File: datasets/dbrd/README.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+### Personal and Sensitive Information
+
+The book reviews and ratings are publicly available on [Hebban](https://www.hebban.nl) and no personal or otherwise sensitive information is contained in this dataset.
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+While prediciting sentiment of book reviews in itself is not that interesting on its own, the value of this dataset lies in its capability for benchmarking models. The dataset contains some challenges that are common to outings on the internet, such as spelling mistakes and other errors. It is therefore very useful to for validating models for their real-world performance. These datasets are abundant for English, but are harder to find for Dutch, making it a valuable resource for ML tasks in this language.
```
Comment (lhoestq) [inline datasets/dbrd/README.md:134]:
```suggestion
While prediciting sentiment of book reviews in itself is not that interesting on its own, the value of this dataset lies in its capability for benchmarking models. The dataset contains some challenges that are common to outings on the internet, such as spelling mistakes and other errors. It is therefore very useful for validating models for their real-world performance. These datasets are abundant for English, but are harder to find for Dutch, making it a valuable resource for ML tasks in this language.
```
typo

**Review by lhoestq — APPROVED** [review]
Thanks for adding these infos ! it's really valuable

**Pull request merged**